test/unit/liquid/tags/debug_test.rb

```diff
@@ -3,7 +3,8 @@
 class Liquid::Tags::DebugTest < ActiveSupport::TestCase
 
   def setup
-    @debug = Liquid::Tags::Debug.parse('debug', ':help', [], {})
+    template = Liquid::Template.parse('{% debug :help %}')
+    @debug = template.root.nodelist.first
```

Comment:
In this and other tests: why not adding a type assertion like you did in the `CdnAsset` test?

`assert_instance_of Liquid::Tags::Debug, tag`